src/gateways/createUserLoader.spec.ts

```diff
@@ -28,36 +28,31 @@ describe('userLoader', () => {
         (global as any).fetch = mockFetch(200, testAccountsUser);
 
         const user = await userLoader.getCurrentUser();
-        expect(fetch).toHaveBeenCalledWith('url/api/user', {
+        expect(fetch).toHaveBeenCalledWith('url/accounts/api/user', {
```

Comment:
Different behavior here (browserAuthProvider always appends `/accounts`) so we'll have to remove `/accounts` from Rex urls to avoid `/accounts/accounts` (or change browserAuthProvider to, for example, not append `/accounts` if the url already ends in `/accounts`

Reply:
this is fine... i think... but don't you have to update some configs because of this?

Reply:
Yeah, I'll go hunt down the configs